src/app/client/src/app/modules/dashboard/dashboard.module.ts

```diff
@@ -17,7 +17,7 @@ import {
 import {
   OrganisationComponent, CourseConsumptionComponent, CourseProgressComponent, UsageReportsComponent,
   DataTableComponent, DataChartComponent, ReportComponent, ReportSummaryComponent, ListAllReportsComponent,
-  AddSummaryModalComponent
+  AddSummaryModalComponent, CourseDashboardComponent, ReIssueCertificateComponent
```

Comment:
These Components should be lazy loaded. Only on clicking on the tabs, then only load the components.